examples/nft/scripts/test.sh

```
@@ -2,18 +2,16 @@
 
 set -e
 
-if [ "$1" = "localnet" ]; then
-  npx hardhat localnet --exit-on-error & sleep 10
-fi
+if [ "$1" = "localnet" ]; then npx hardhat localnet --exit-on-error & sleep 10; fi
 
-function nft_balance() {
+function balance() {
   local ZETACHAIN=$(cast call "$CONTRACT_ZETACHAIN" "balanceOf(address)(uint256)" "$SENDER")
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)